libs/experimental/tests/integration_tests/chains/test_sql_database.py

```diff
@@ -77,7 +77,7 @@ def test_sql_database_sequential_chain_intermediate_steps() -> None:
     db_chain = SQLDatabaseSequentialChain.from_llm(
         OpenAI(temperature=0), db, return_intermediate_steps=True
     )
-    output = db_chain("What company does Harrison work at?")
+    output = db_chain.run("What company does Harrison work at?")
```

Comment:
We generally prefer using .invoke rather than .run now

Reply:
@eyurtsev, Understood, I have updated PR with .invoke now with all the found code that should be modified with the same.
Thanks

Reply:
@eyurtsev, I also see that a lint error comes up with this change. It seems like .invoke is an expected nonstring data type value, maybe a dict type.

<img width="1037" alt="image" src="https://github.com/langchain-ai/langchain/assets/141953346/0c2c1c99-2529-4875-884c-2a8393398a9a">

Reply:
@eyurtsev, Just a reminder, Should we revoke from **.invoke** to **.run** now?